manifests/tasks/object-name-sanitise.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: object-name-sanitise
+spec:
+  description: This Task can be used to sanitise a string that will be used for naming a k8s object. This task converts the string to a value that can be used for an object. 
+  params:
+  - name: input-string
+    type: string
+  results:
+  - name: output-string
+    description: Total size of the model files
+  steps:
+  - name: sanitise-input-string
+    image: registry.access.redhat.com/ubi9/ubi-minimal
+    script: |
+      #!/usr/bin/env bash
+
+      set -Eeuo pipefail
+
+      echo $(params.input-string) | sed 's/[^a-z0-9.-]/-/g; s/-*$//; s/\n*$//; s/ *$//' | tr -d "\n"  > $(results.output-string.path);
+
+    workingDir: /
```

Comment:
For brevity I would even remove this.

Reply:
Which part the working dir?

Reply:
@jackdelahunt oh, my bad, I meant `workingDir: /`

Reply:
No problem